homeassistant/components/jewish_calendar/sensor.py

```diff
@@ -226,9 +222,7 @@ def make_zmanim(self, date: Date) -> Zmanim:
     @property
     def extra_state_attributes(self) -> dict[str, str]:
         """Return the state attributes."""
-        if self.entity_description.key != "holiday":
-            return {}
-        return self._holiday_attrs
+        return {} if self.entity_description.key != "holiday" else self._holiday_attrs
```

Comment:
I wouldn't change this syntax, as we might want to have different attributes for different sensors. Currently thinking about Omer, adding days/weeks